sos/plugins/ovn_central.py

```diff
@@ -57,20 +76,39 @@ def setup(self):
 
         # Some user-friendly versions of DB output
         cmds = [
+            'ovn-nbctl show',
+            'ovn-sbctl show',
             'ovn-sbctl lflow-list',
             'ovn-nbctl get-ssl',
             'ovn-nbctl get-connection',
             'ovn-sbctl get-ssl',
             'ovn-sbctl get-connection',
         ]
 
+
```

Comment:
Please remove this extra empty line (Travis fails on it - you can run "pycodestyle" on modified code to spot it locally before committing it).